superset/assets/src/dashboard/stylesheets/variables.less

```diff
@@ -16,62 +16,45 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-@indicator-color: #44C0FF;
+@indicator-color: #44c0ff;
 
 @almost-black: #263238;
 @gray-dark: #484848;
 @gray: #879399;
-@gray-light: #CFD8DC;
+@gray-light: #cfd8dc;
 @gray-bg: #f5f5f5;
-@gray-heading: #A3A3A3;
-@menu-hover: #F2F3F5;
+@gray-heading: #a3a3a3;
+@menu-hover: #f2f3f5;
 
 /* builder component pane */
 @builder-pane-width: 374px;
 
 /* toasts */
-@pink: #E32364;
-@purple: #2C2261;
+@pink: #e32364;
+@purple: #2c2261;
 
-@success: #00BFA5;
-@warning: #FFAB00;
+@success: #00bfa5;
+@warning: #ffab00;
 @danger: @pink;
 
 /* filter indicators */
 /* make sure be consistent with FILTER_COLORS_COUNT in
    dashboardFiltersColorMap.js
 */
-@badge-colors:
-  #228be6,
-  #40c057,
-  #fab005,
-  #f76707,
-  #e64980,
-  #15aabf,
-  #7950f2,
-  #fa5252,
-  #74b816,
-  #12b886,
-  #1864ab,
-  #2b8a3e,
-  #e67700,
-  #d9480f,
-  #a61e4d,
-  #0b7285,
-  #5f3dc4,
-  #c92a2a,
-  #5c940d,
-  #087f5b;
+@badge-colors: #228be6, #40c057, #fab005, #f76707, #e64980, #15aabf, #7950f2, #fa5252, #74b816,
+  #12b886, #1864ab, #2b8a3e, #e67700, #d9480f, #a61e4d, #0b7285, #5f3dc4, #c92a2a, #5c940d, #087f5b;
 
 @iterations: length(@badge-colors);
 .badge-loop (@i) when (@i > 0) {
   .filter-badge.badge-@{i},
   .active .color-bar.badge-@{i},
   .dashboard-filter-indicators-container:hover .color-bar.badge-@{i},
   .dashboard-component-chart-holder:hover .color-bar.badge-@{i} {
-    @value:  extract(@badge-colors, @i);
+    @value: extract(@badge-colors, @i);
     background-color: @value;
   }
   .badge-loop(@i - 1);
 }
 .badge-loop (@iterations);
+
```

Comment:
Can we prefix this new section with a comment? something describing that z index variables go here now?